nilearn/datasets/tests/test_struct.py

```diff
@@ -157,7 +157,7 @@ def test_fetch_icbm152_brain_gm_mask(tmp_path, request_mocker):
     assert isinstance(grey_matter_img, nibabel.Nifti1Image)
 
 
-@pytest.mark.parametrize("mesh", ["fsaverage5", "fsaverage"])
+@pytest.mark.parametrize("mesh", ["fsaverage5", "fsaverage", "fsaverage6"])
```

Comment:
We need to update the test a little bit to make it work.
I don't know if the plan is to ship fsaverage6 with nilearn as we are currently doing with fsaverage5 or not.
If we don't ship it, we need to mock the download of fsaverage6 in this test in the same way as for fsaverage. Once the structure of the files will be clear (see previous comments), we should be able to mock the download by adding an entry "*fsaverage6.tar.gz" in the `url_mapping` dict of the mocker. 
Does this make sense? (also pinging @jeromedockes here who probably has an opinion on this).

Reply:
yes exactly.
fsaverage6 would not be distributed with nilearn, but be downloaded and the test needs to mock that as you explain.
I think the plan of @alexisthual  was also to add other fsaverage meshes such as fsaverage4; if there is a very low-resolution one (significantly smaller than fsaverage5), then we could ship that one (in addition to fsaverage5) with nilearn and use it in unit tests instead of fsaverage5

Reply:
IMHO fsaverage6 is actually not that big. Wouldn't it make our life easire to ship it too ?